test/integration/http_subset_lb_integration_test.cc

```diff
@@ -176,7 +176,10 @@ class HttpSubsetLbIntegrationTest
       }
     }
 
-    if (is_hash_lb_) {
+    // The default number of choices for the LEAST_REQUEST policy is 2 hosts, if the number of hosts
+    // is equal to the number of choices, a full scan happens instead, this means that the same host
+    // will be chosen.
+    if (is_hash_lb_ || (GetParam() == envoy::config::cluster::v3::Cluster::LEAST_REQUEST)) {
```

Comment:
Note to self: check whether this patch from the original enable full scan PR is actually still needed

Reply:
Removing. No longer applicable, since full scan mode will require opt-in in all cases